checker/tests/tainting/TaintingPolyFields.java

```diff
@@ -3,6 +3,7 @@
 import org.checkerframework.checker.tainting.qual.Tainted;
 import org.checkerframework.checker.tainting.qual.Untainted;
 
+// :: error: (missing.annotation.on.class)
```

Comment:
I expected this error to be issued at the field rather than the class. Then the error would be something like ‘Cannot use a polymorphic qualifier on a field unless the class is annotated with @HasQualifierParameter. (Field declarations are VariableTrees. )